pytest_fixtures/satellite_auth.py

```diff
@@ -227,6 +227,7 @@ def enroll_configure_rhsso_external_auth():
     run_command(cmd="systemctl restart httpd")
 
 
+@pytest.mark.external_auth
```

Comment:
Same here, these need to stay on the tests for now. We've got to use hooks to apply them through the fixtures.